web/pages/GroupDetails.tsx

```
@@ -1,712 +1,927 @@
+                        {loading ? Array(3).fill(0).map((_, i) => <Skeleton key={i} className="h-24 w-full rounded-2xl" />) :
+                            expenses.map((expense, idx) => (
+                                <motion.div
+                                    layout
+                                    key={expense._id}
+                                    initial={{ opacity: 0, scale: 0.95 }}
+                                    animate={{ opacity: 1, scale: 1 }}
+                                    whileHover={{ scale: 1.02 }}
+                                    onClick={() => openEditExpense(expense)}
+                                    onKeyDown={(e) => { if (e.key === 'Enter' || e.key === ' ') { e.preventDefault(); openEditExpense(expense); } }}
+                                    tabIndex={0}
+                                    role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)